boreas/ping.c

```diff
@@ -197,6 +197,7 @@ send_icmp_v4 (int soc, struct in_addr *dst)
   icmp = (struct icmphdr *) sendbuf;
   icmp->type = ICMP_ECHO;
   icmp->code = 0;
+  icmp->un.echo.sequence = 0x0100;
```

Comment:
Should we also populate id?

I'm not a C developer but the value of sendbuf is undefined right?
```
struct icmphdr
{
  u_int8_t type;                /* message type */
  u_int8_t code;                /* type sub-code */
  u_int16_t checksum;
  union
  {
    struct
    {
      u_int16_t        id;
      u_int16_t        sequence;
    } echo;                        /* echo datagram */
    u_int32_t        gateway;        /* gateway address */
    struct
    {
      u_int16_t        __unused;
      u_int16_t        mtu;
    } frag;                        /* path mtu discovery */
  } un;
};
```

Reply:
I wonder why in our case it was always zero? Are older kernels leaving it as is?
In 
```Linux <HOSTNAME> 5.4.0-148-generic #165-Ubuntu SMP Tue Apr 18 08:53:12 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux``` we were seeing the 

```
SENT (1.0342s) ICMP [192.168.1.30 > 192.168.1.1 Echo request (type=8/code=0) id=0 seq=0] IP [ttl=64 id=25987 iplen=28 ]
```
id=0 for ipv4